.bcr/config.yml

```diff
@@ -0,0 +1,4 @@
+# see https://github.com/bazel-contrib/publish-to-bcr/#a-note-on-release-automation
+fixedReleaser:
+  login: thesayyn
+  email: sahin@aspect.dev
```

Comment:
Can these be set to some team email?  At the very least it should be one of us

Reply:
Sure, can you propose an edit of who on your team plans to respond to emails about publishing failures to the BCR?

Reply:
unless we can put a group here, let's just remove this file?  I don't think we want a fixed author if it has to be an individual

Reply:
Sure, I can remove this file, but depending on how you create releases, it may fail to publish.

Publish to BCR uses information about the GitHub author of a release in order to tag that author in the commit, push an entry to their BCR fork, or send error notifications. If you use a GitHub action to automate your release and the author is the `github-actions` bot (likely because you used the `GITHUB_TOKEN` secret to authorize the action), then the app won't know who cut the release and may not function properly.

Reply:
see https://github.com/bazel-contrib/publish-to-bcr/blob/262c9e15a3c556cb6ecc7ce67e31956acf91e201/README.md#a-note-on-release-automation

Reply:
fair enough, I guess it's an arbitrary choice anyway.  Let's just go with one of the names below?

Reply:
I'll just remove the file for this pass, it's possible if you're creating manual GH releases that the releaser identity will propagate correctly.